HANDS_ON_LAB.md

```diff
@@ -745,19 +745,7 @@ oc start-build cloud-native-backend-$(oc project -q)
 ```
 
 - Open your project within the OpenShift console and select `Pipelines` under the `Build` screen
-- Look to your pipeline created and check if the build has been started
```

Comment:
We must keep `Look to your pipeline created and check if the build has been started` line

Reply:
Hmhmhm. I'm wondering why our users with admin role can use the `jenkins` server ....

Reply:
I can change it of course, but I believe that the instructions are pretty clear when we say:

* Open your project within the OpenShift console and select Pipelines under the Build screen
* Wait until the pipeline has been executed

Reply:
I don't really know why the admin role is needed in order to access Jenkins. I would probably have to check the code to figure that out since there is no relevant setting in the Openshift `master-`config.yml